src/content/cloud/deploy-from-helm.mdx

```diff
@@ -10,6 +10,9 @@ import Image from '@theme/Image';
 You can launch additional services for your development environments on Okteto Cloud using Helm repositories.
 Okteto automates all the processes needed to deploy, configure, upgrade, and destroy your Helm releases, so you can focus on building amazing applications.
 
+You can also list your own charts under the [Helm Catalog](/docs/enterprise/administration/configuration/#applications) tab of the **Launch Dev Environment** menu.
```

Comment:
Can we add a subheader for this? (e.g. "Add your own charts") and the TierList badge for 'Self-hosted'?